x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_exploration/components/regression_exploration/evaluate_panel.tsx

```diff
@@ -139,35 +258,40 @@ export const EvaluatePanel: FC<Props> = ({ jobConfig, jobStatus }) => {
               )}
             </span>
           </EuiTitle>
+          {generalizationDocsCount !== null && (
```

Comment:
Do we have the count of docs in the main table? If so, would be good to remove the 'first 1000 docs' message above the table if there are less than 1000 docs:

![image](https://user-images.githubusercontent.com/7405507/69256087-d4c43700-0bb0-11ea-947c-f726d488e041.png)

Reply:
Updated in 494ac87820e7fc2863e03dc244f44c9a290bff4c